File: Machine/Models/Blank/CoinBlank.cs
using System;
using Machine.Models.Enum;

namespace Machine.Models.Blank
{
	public class CoinBlank
	{
		public Int32 Id { get; set; }
		public CoinCategory CoinCategory { get; set; }
		public Int32 MaxCoinCount { get; set; }
		public Int32 NowCoinCount { get; set; }
	}
}

File: Machine/ClientApp/src/Blank/coinBlank.ts
import {CoinCategory} from "../Enum/CoinCategory";

export class CoinBlank{
	public id: number;
	public coinCategory: CoinCategory;
	public maxCoinCount: number;
	public nowCoinCount: number;
	constructor(id: number, coinCategory: CoinCategory, maxCoinCount: number, nowCoinCount: number) {
		this.id = id;
		this.coinCategory = coinCategory;
		this.maxCoinCount = maxCoinCount;
		this.nowCoinCount = nowCoinCount;
	}
}